options.go

```diff
@@ -91,6 +91,7 @@ func RequestHeader(header string) Option {
 func FieldName(name string) Option {
 	return func(cs *csrf) error {
 		cs.opts.FieldName = name
+		fieldName = name
```

Comment:
@molivier 

RE: `fieldName` as the map key—you're right. Add a const that sets the context key to `goji.csrf.FieldName` - with the default fieldname still reverting to `tokenKey` ("goji.csrf.Token").